### ISSUE TITLE
Feature request: add an option to disable stats snapshots

### ISSUE THREAD

**Issue by gjethwani** [nodeshift/opossum#798]
Hello! I had a small feature request

**Motivation**

As I understand it, every `bucket_interval` (1s by default), opossum emits a snapshot of the stats in the `Status` class

```
this[SNAPSHOT_INTERVAL] = setInterval(_ => this.emit('snapshot', this.stats), bucketInterval);
```

My app doesn't actually listen to these stats and the extra call to `setInterval` clogs up the memory

**Solution**

Could we have a configurable option to disable the stats snapshots, thereby removing the extra call to `setInterval`?

**Comment by gjethwani** [comment]
Created a PR here: https://github.com/nodeshift/opossum/pull/799

**Comment by gjethwani** [comment]
Corresponding PR in DefinitelyTyped: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66220

**Comment by lholmquist** [comment]
just published 8.1.0
